docs/usage.rst

```diff
@@ -200,3 +200,99 @@ This is the case when you choose to integrate your user registration process wit
 Otherwise, serializer classes for ``verify`` and ``register`` views will not be available.
 
 3. Latest ``security_code`` generated for a ``phone_number`` can be found at ``/admin/phone_verify/smsverification/`` URL.
+
+
+Case 3: Create a user model and login the user once their phone number gets verified
+*************************************************************************************
+
+This case is suitable if you want to allow users to login to your django project using its amazing authentication backend.
+**Note**: The code below assumes you are using Django Templating system. The same logic would apply for a rest API.
+
+.. code-block:: python
+
+
+    # In models.py
+    from django.db import models
+    from django.contrib.auth.models import AbstractUser
+
+    # Create a custom user model by extending AbstractUser so that all the authetication functionality can be retained
+    class User(AbstractUser):
+        # Additional User fields can be added in here
+        phone_number = models.CharField(null=True, blank=True, max_length=20, unique=True)
+
+
+
+
+    #  In views.py
+    from django.contrib.auth import get_user_model, login
+    from django.http import HttpResponse
+
+    #  Custom Login view for users that are logging in using verified phone numbers
+    def LoginOTPUser(request):
+        phone_number = request.POST['phone_number']
+
+        try:
+            # Get the associated User model instance
+            user = get_user_model().objects.get(phone_number=phone_number)
+
+            # Login this user
+            login(request, user, 'django.contrib.auth.backends.ModelBackend')
+
+            return HttpResponse(
+                dumps({
+                    'status': '200',
+                }), content_type="application/json"
+            )
+        except Exception as e:
```

Comment:
Not a good way of handling exceptions. If there are exceptions, it should be explicit on what exceptions and figure out what was done and why.

Also, please don't use print :)